e2e/e2e.sh

```diff
@@ -65,9 +68,14 @@ pack() {
 }
 
 run_docker_dbeam() {
+  OUTPUT_MOUNT_EXP=""
+  if [ "$MOUNT_OUTPUT" -eq 1 ]; then
+    OUTPUT_MOUNT_EXP=--mount="type=bind,source=$SCRIPT_PATH,target=$SCRIPT_PATH"
+  fi
   time docker run --interactive --rm \
     --net="$DOCKER_NETWORK" \
     --mount="type=bind,source=$PROJECT_PATH/dbeam-core/target,target=/dbeam" \
+    $OUTPUT_MOUNT_EXP \
```

Comment:
```suggestion
    --mount="type=bind,source=$SCRIPT_PATH,target=$SCRIPT_PATH" \
```

I guess it can be simply this. Any downsides?

Reply:
I don't know if output was intentionally prevented from getting to the host machine. I would make output available on host all the time, it's really useful to examine those files

Reply:
It is not. It can always be mounted. The suggestion applies and MOUNT_OUTPUT can be removed.